crates/polars-ops/src/chunked_array/strings/flarion_split.rs

```diff
@@ -0,0 +1,181 @@
+use arrow::array::ValueSize;
+use regex::RegexBuilder;
+
+use super::*;
+
+// I'm hoping the compiler optimizes this out, but it is required for correct outputs
+#[inline]
```

Comment:
#[inline(always)] ? 😭

Reply:
LLVM pretty much straight up ignores inline(always), I was actually more referring to the collect() and then into_iter() calls^^